From b8d8406f6dff91f328c41a26998536d9fe63919a Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Wed, 30 Aug 2017 18:10:24 +0800
Subject: [PATCH 2471/5242] MLK-16298-2 ARM64: imx8mq-evk: typec super speed
 mux select gpio update

commit  bf4c087d415668d22297b2987dea683c9a14b11c from
https://source.codeaurora.org/external/imx/linux-imx.git

Use GPIO_ACTIVE_HIGH for super speed signal mux selection GPIO as
we should pull it up to be high for CC1 according to PTN36043.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 5479e06..4e15a63 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -429,7 +429,7 @@
 		reg = <0x50>;
 		interrupt-parent = <&gpio3>;
 		interrupts = <3 8>;
-		ss-sel-gpios = <&gpio3 15 GPIO_ACTIVE_LOW>;
+		ss-sel-gpios = <&gpio3 15 GPIO_ACTIVE_HIGH>;
 		src-pdos = <0x380190c8>;
 		snk-pdos = <0x380190c8 0x3802d0c8>;
 		max-snk-mv = <9000>;
-- 
1.7.9.5

